scripts/start-ranger-usersync.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+source /ranger-common.sh
+
+cd ${RANGER_USERSYNC_HOME}
+
+[[ -z $rangerUsersync_password ]] && export rangerUsersync_password=$(vault read -field=rangerUsersync_password ${vault_path}/ranger_admin)
```

Comment:
Why is this environment variable lower case while the others are upper?

Reply:
we are just setting properties required in install.properties as Environment variables and running following to replace and run setup.sh
[[ -n ${!ini} ]] && sed -i "s/^${ini}=.*$/${ini}=${!ini}/" install.properties

if we need to customize names, we need to instead use a case statement.

Reply:
Ah, OK, so this is a "standard" Ranger environment variable not one of our own so we don't have control over it. Fine, then we should leave it as is.

Reply:
resolving conversation based on above...